docs/cli-guide/master-docinfo.xml

```diff
@@ -1,9 +1,9 @@
-<title>Using the {ProductShortName} command-line interface to analyze applications</title>
+<title>Managing applications with the {ProductShortName} command-line interface</title>
```

Comment:
Considering that the main functionality of MTA is to help organizations migrate and modernize their applications for cloud and container platforms, I'd suggest updating the title to 'Modernizing and migrating applications using the MTA CLI' .